### DISCUSSION TITLE
Enable payment method change in WooSubscription

### DISCUSSION THREAD

**Discussion by jdslv** [woocommerce/woocommerce-gutenberg-products-block#11552] [Q&A - Extension Integrations and Customizations]
Hello,

It seems not the good channel to ask for help but I was asked to leave a message here as Automattic does "not have dedicated payment / subscription support".

I am adding support on WooSubscription on a payment gateway. 

The documentation says ["Subscriptions creates a mock checkout using the original order details and overriding the total to be $0."](https://woo.com/document/subscriptions/develop/payment-gateway-integration/#section-18), I tried to add the needed flag (`subscription_payment_method_change`) but I never manage to have a new order, the change button does nothing.

Anyone can give me a hand to unblock this situation?

Thanks

**Comment by opr** [comment]
Hi, you're right this isn't really the best channel to discuss this as this repo is for the WooCommerce Blocks extension.

I'd recommend joining the [WooCommerce Community Slack](https://woo.com/community-slack/) and asking your question there, hopefully it would get a little more feedback from people who may be able to help.

You may also want to open an issue on https://github.com/automattic/woocommerce-subscriptions-core where folks who work on that extension may be able to help. 

I'll close this issue as it's not actionable from the WooCommerce Blocks team. Thank you.
